jest.config.js

```diff
@@ -4,8 +4,8 @@ module.exports = {
         "^.+\\.tsx?$": 'ts-jest'
     },
     testPathIgnorePatterns: [
-        "/build/",
-        "/node_modules/",
+        "<rootDir>/build/",
+        "<rootDir>/node_modules/",
```

Comment:
This change is required or the tests could not be resolved in Travis CI. The cause is that Travis CI put the project in a `/home/travis/build/` directory, which will be ignored by Jest. The `<rootDir>` token is the standard way to avoid this issue.